build.sh

```diff
@@ -1,8 +1,25 @@
 #!/bin/bash
 
 mkdir -p ./build
```

Comment:
this line can be deleted as the `-p` in the next lines already creates the parent folders

Reply:
Line was removed.